Formula/pueue.rb

```diff
@@ -0,0 +1,66 @@
+class Pueue < Formula
+  desc "Command-line tool for managing long-running shell commands"
+  homepage "https://github.com/Nukesor/pueue"
+  url "https://github.com/Nukesor/pueue/archive/v0.1.4.tar.gz"
+  sha256 "8e156d1037a88448219d9c5bdbc844799e616604985c86a83541f0a23481f302"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+    rm bin/"completions_pueue"
+
+    prefix.install_metafiles
+
+    bash_completion.install "utils/completions/pueue.bash" => "pueue.bash"
+    fish_completion.install "utils/completions/pueue.fish" => "pueue.fish"
+  end
+
+  plist_options :manual => "pueued"
```

Comment:
I think this should be `pueued -d` for the purposes described here.

Reply:
From what I understand the `-d` option forks the process. Using the `plist` as a supervisor this shouldn't be necessary.
The way I see this working is using `pueued -d` if I don't want to start the service with `brew services {run,start} pueue`, for example running only once.
I'd leave it running in the foreground and letting `launchd` do the supervision.

Reply:
Can confirm. Only use the `-d` flag, if you don't want `pueued` to be run by a service supervisor.